README.md

```diff
@@ -5,16 +5,13 @@
 
 Choose a design for your landing page:
 - [MET landing](https://www.figma.com/file/lSR1m42L9YwzQwzzxKwHpw/THE-MET)
-- [MYBIKE landing](https://www.figma.com/file/NZQAIydtHo5QkINyGLHNcq/BIKE-New-Version?node-id=0%3A1)
-- [B&O](https://www.figma.com/file/DtkQmQ797hk0nI4KfMi2Uq/BOSE-New-Version?type=design&node-id=6817-212&t=ZTV6Gl8NzaWkJ4FK-0)
-- [Nothing](https://www.figma.com/file/DtkQmQ797hk0nI4KfMi2Uq/BOSE-New-Version?type=design&node-id=6802-139&t=L7eKz5YKLN0m5WxR-0)
 
 Follow the videos in this and next lessons to implement the page block by block
 - Here is [the design from the video](https://www.figma.com/file/DtkQmQ797hk0nI4KfMi2Uq/BOSE-New-Version?type=design&node-id=6703-88&t=L7eKz5YKLN0m5WxR-0) (❗❗ DO NOT implement this design ❗❗ choose one of the options above ❗❗ ☝️)
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://nnalinn.github.io/layout_landing-page/)
```

Comment:
Add this demo link to the PR description 
<img width="967" alt="image" src="https://github.com/user-attachments/assets/1859a2ad-04dc-4355-9437-66bbdd551b14" />